index.html

```diff
@@ -93,6 +93,9 @@
               <li>
                 <a href="/colorado" class="stateNav-colorado">Colorado</a>
               </li>
+              <li>
+                <a href="/oregon" class="stateNav-oregon">Oregon</a>
```

Comment:
you don't really need to make edits to this file, it's not being used in the webpack build

Reply:
oh my mistake - thx